web/templates/navbar.hbs

```diff
@@ -35,7 +35,7 @@
                             <div id="search_arrows" class="input-append">
                                 <span class="search_icon search_open" ><i class="zulip-icon zulip-icon-search"></i></span>
                                 <input class="search-query input-block-level home-page-input" id="search_query" type="text" placeholder="{{t 'Search' }}"
-                                  autocomplete="off" aria-label="{{t 'Search' }}" title="{{t 'Search' }} (/)"/>
+                                  autocomplete="off"/>
```

Comment:
Did someone mention that we should remove this ?

Reply:
Since there was no discussion about this particular tooltip, I included it in the PR for feedback. Should I remove the bootstrap tooltip, given that the tippy tooltip with the same content is already displayed on the `search_closed` icon?

Reply:
<img width="930" alt="Screenshot 2023-04-16 at 5 30 40 PM" src="https://user-images.githubusercontent.com/25124304/232308400-6cb9457c-5430-4abd-8404-f6152aaa2868.png">

@alya this code triggers a default browser tooltip which appears on a long hover on the search bar. I think It doesn't hurt to keep this, since the search bar is quite wide. What do you think?

Reply:
> Since there was no discussion about this particular tooltip, I included it in the PR for feedback. 

@Ddharmani3 If you are including something for feedback, it is essential to call out the question in the PR description. Certainly, I will never know that there is such a question if it's only revealed by a careful review of the code.

Reply:
This is noted specifically in the PR template:

>  Calls out remaining decisions and concerns.

Reply:
Let's have a patter of always using the modern Tippy styling for tooltips that include keyboard shortcuts. So we can replace this long-hover tooltip with a Tippy one.